rasa/shared/nlu/training_data/training_data.py

```diff
@@ -607,7 +606,7 @@ def _training_examples_without_empty_e2e_examples(self) -> List[Message]:
         return [
             example
             for example in self.training_examples
-            if not example.get(ACTION_NAME) and not example.get(INTENT_NAME)
+            if not example.get(ACTION_NAME) and not example.get(INTENT)
```

Comment:
this condition breaks things

Reply:
yes, just pushed the fix with correct condition